prov/gni/src/gnix_fabric.c

```diff
@@ -363,14 +366,19 @@ static int gnix_getinfo(uint32_t version, const char *node, const char *service,
 				goto err;
 			}
 
-			if (hints->ep_attr->tx_ctx_cnt > 1) {
+			if (hints->ep_attr->tx_ctx_cnt > GNIX_SEP_MAX_CNT) {
```

Comment:
From the man page:
As a general rule, specifying a non-zero  value  for  input hints indicates that a provider must support the requested value or fail the operation with -FI_ENO-DATA.